index.html

```diff
@@ -953,6 +953,9 @@ <h1>How Can I Join Projects</h1>
 								<li><a target="_self" href="https://www.linkedin.com/groups/FOSSASIA-3762811"><i class="icon social_linkedin icon-large"></i></a></li>
 								<li><a target="_self" href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ"><i class="icon social_youtube icon-large"></i></a></li>
 								<li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_instagram icon-large"></i></a></li>
+								<li><a target="_self" href="https://github.com/fossasia"><i class="icon social_github icon-large"></i></a></li>
+								<li><a target="_self" href="https://plus.google.com/108920596016838318216"><i class="icon social_google icon-large"></i></a></li>
```

Comment:
The google social icon doesn't exist. Try testing it.

Replace both icons with valid one.